### PR TITLE
React 15 update

### PR DESCRIPTION
**Pull request by jhen0409**
Currently it has `peerinvalid` errors in the [on-debugger build](https://travis-ci.org/jhen0409/remote-redux-devtools-on-debugger/jobs/127546345#L317) for npm@2.

Also, the `react-switcher` still have `react@0.14` dependency, @zalmoxisus maybe you should use `^0.14.0 || ^15.0.0` as a peer dependency?

### CODE REVIEW

**Comment by zalmoxisus** [comment]
If the problem is only with `react-switcher`, I'll update it. Thanks!

**Comment by zalmoxisus** [comment]
Oh, should we have `"react": "^15.0.2"` not `"react": "^0.15.2"`?

File: package.json [hunk 1]
```diff
@@ -83,12 +83,12 @@
   "dependencies": {
     "chrome-storage-local": "^0.1.5",
     "jsan": "^3.1.2",
-    "react": "^0.14.5",
-    "react-addons-shallow-compare": "^0.14.8",
+    "react": "^15.0.2",
```
Comment (zalmoxisus) [inline package.json]: I guess you have a typo here, don't you?

Comment (jhen0409) [inline package.json]: It's indeed `^15.0.2`.

Comment (zalmoxisus) [inline package.json]: Indeed, I see. Strange O_o.

**Pull request merged**